src/net.md

```diff
@@ -1123,6 +1124,99 @@ fn run() -> Result<()> {
 # quick_main!(run);
 ```
 
+[ex-rate-limit-exceeded]: #ex-rate-limit-exceeded
+<a name="ex-rate-limit-exceeded"/>
+## Handling the Github rate limit error condition
+
+[![reqwest-badge]][reqwest] [![hyper-badge]][hyper] [![cat-net-badge]][cat-net]
+
+This example uses the [GitHub API - Rate limiting], as an example of how to handle error conditions.
+
```

Comment:
I think it might be helpful to elaborate a little bit on what is going on in this example. It doesn't need to be an essay, but maybe just briefly spelling out the steps or any key points the reader should be aware of.